app/src/main/res/layout/activity_main.xml

```diff
@@ -26,12 +26,25 @@
             android:src="@drawable/icon" />
 
         <Button
-            android:id="@+id/newGame"
+            android:id="@+id/resumeGame"
```

Comment:
ids in XML files should be in `snake_case` just as a convention. Though when using Kotlin you can use the ids in camel case as well. 
Reference: https://code.i-harness.com/en/q/1bf562